core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -22,55 +25,62 @@ import scala.util.Try
  */
 class HttpExecutionServiceDAO( executionServiceURL: String, submissionTimeout: FiniteDuration )( implicit val system: ActorSystem ) extends ExecutionServiceDAO with DsdeHttpDAO with Retry with FutureSupport with LazyLogging {
 
+  //sendReceive, but with gzip.
+  //Two versions, one of which overrides the timeout.
```

Comment:
(and presumably doing so here would override us setting the default timeout from config)